crm_claim_rma/models/crm_claim.py

```diff
@@ -147,6 +147,15 @@ def __create_lines_from_invoice(self):
         for invoice_line in invoices_lines:
             location_dest = claim_line.get_destination_location(
                 invoice_line.product_id, warehouse)
+            line = self.__create_line_from_invoice_line(
+                invoice_line,
+                location_dest
+            )
+            lines.append((0, 0, line))
+        return lines
+
+    def __create_line_from_invoice_line(self, invoice_line, location_dest):
```

Comment:
Please call the function something like `_prepare_claim_line_from_invoice_line` since it's preparing a dict to create a line and not creating a line.